primitives/src/tokens.rs

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Unnecessary.

Reply:
No not relevent to nodle context, clean-up done :+1: